proto/greenfield/virtualgroup/params.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+package greenfield.virtualgroup;
+
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/bnb-chain/greenfield/x/virtualgroup/types";
+
+// Params defines the parameters for the module.
+message Params {
+  option (gogoproto.equal) = true;
+  option (gogoproto.goproto_stringer) = false;
+
+  // deposit_denom defines the staking coin denomination.
+  string deposit_denom = 1;
+
+  // store price, in bnb wei per charge byte
+  string gvg_staking_price = 2 [
```

Comment:
Do we need to make it as a versioned parameter? If this parameter changes, when `Withdraw` , can we do the correct calculation?